src/oneD/refine.cpp

```diff
@@ -12,7 +12,7 @@ namespace Cantera
 {
 Refiner::Refiner(Domain1D& domain) :
     m_ratio(10.0), m_slope(0.8), m_curve(0.8), m_prune(-0.001),
-    m_min_range(0.01), m_domain(&domain), m_npmax(3000),
+    m_min_range(0.01), m_domain(&domain), m_npmax(300),
```

Comment:
I think 1000 would be fine. I don't think it makes any sense for for that argument of `Sim1D::setMaxGridPoints` to have a default value.